db/migrate/20180924071722_add_translate_pages.rb

```diff
@@ -0,0 +1,15 @@
+class AddTranslatePages < ActiveRecord::Migration
+  def self.up
+    SiteCustomization::Page.create_translation_table!({
+      title: :string,
+      subtitle: :string,
+      content: :text
+    })
+    change_column :site_customization_pages, :title, :string, :null => true
+  end
+
+  def self.down
+    SiteCustomization::Page.drop_translation_table!
+    change_column :site_customization_pages, :title, :string, :null => false
+  end
+end
```

Comment:
I think this missing newline character is the only spacing issue left.

Reply:
Yes, sorry. Updated.